### ISSUE TITLE
xef-gcp - Runtime

### ISSUE THREAD

**Issue by jackcviers** [xebia-functional/xef#291]
Using the api overview (https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/overview) and the available embeddings, create a xef gcp runtime similar to that for xef-openai that sets sensible defaults, initializes the client, and implements the client dsl.

Depends on  #290

**Comment by Intex32** [comment]
Duplicate of #371 ?

**Comment by anamariamv** [comment]
Solved #371
